include/fmt/core.h

```diff
@@ -58,10 +58,12 @@
 // Check if relaxed C++14 constexpr is supported.
 // GCC doesn't allow throw in constexpr until version 6 (bug 67371).
 #ifndef FMT_USE_CONSTEXPR
+#if !(defined(__NVCC__) || defined(__CUDACC__))
```

Comment:
Is it necessary to check both `__NVCC__` and `__CUDACC__`? Can one be set and another not?

Reply:
It is not necessary as `__CUDACC__` can be considered a subset of `__NVCC__`. I'll remove the `__CUDACC__` check: https://docs.nvidia.com/cuda/cuda-compiler-driver-nvcc/index.html#nvcc-identification-macro

Reply:
Done.